app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/helpers/SummaryListBuilder.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsoftwarechoicesfrontend.views.helpers
+
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content._
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist._
+import uk.gov.hmrc.http.InternalServerException
+import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.controllers.routes
+import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.pages._
+import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.services.PageAnswersService
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait SummaryListBuilder {
```

Comment:
Could it potentially make more sense here to not pass the service into here and instead pass the answers instead?

I'm seeing 4 separate calls out made to fetch details from mongo which could just as easily be 1.

Won't need to deal with Future then too